src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1971,7 +2000,6 @@ impl DdlController {
         } = actor_graph_builder
             .generate_graph(&self.env, stream_job, expr_context)
             .await?;
-        assert!(dispatchers.is_empty());
```

Comment:
Can we still keep this assertion depending on the table job type?

Reply:
fixed